parsely/src/main/java/com/parsely/parselyandroid/ParselyTrackerInternal.kt

```diff
@@ -6,7 +6,7 @@ import androidx.lifecycle.LifecycleEventObserver
 import androidx.lifecycle.ProcessLifecycleOwner
 import java.util.UUID
 
-internal class ParselyTrackerInternal internal constructor(
+public class ParselyTrackerInternal internal constructor(
```

Comment:
Any chance this is an accidental change? 😓 If not, do you mind explaining why this needs to be public?

Reply:
Yes, definitely accidental change.. thank you for noticing this 🙇